populate/src/main/java/bio/terra/pearl/populate/service/PortalParticipantUserPopulator.java

```diff
@@ -67,4 +74,45 @@ public PortalParticipantUser createPreserveExisting(PortalParticipantUser existi
     public PortalParticipantUser createNew(PortalParticipantUser popDto, PortalPopulateContext context, boolean overwrite) {
         return portalParticipantUserService.create(popDto);
     }
+
+    public List<String> bulkPopulateParticipants(String portalShortcode, EnvironmentName envName, String studyShortcode, Integer numEnrollees) {
+        StudyPopulateContext context = new StudyPopulateContext("portals/" + portalShortcode + "/participants/seed.json", portalShortcode, studyShortcode, envName, new HashMap<>());
+
+        List<String> populatedUsernames = new ArrayList<>();
+
+        IntStream.range(0, numEnrollees).forEach(i -> {
```

Comment:
Your FP is showing. 😂 While you're at it, may as well go all the way and use `collect` to construct your username list:
```
        List<String> populatedUsernames = IntStream.range(0, numEnrollees).mapToObj(i -> {
            try {
                ...
                return username;
            } catch (IOException e) {
                ...
            }
        }).collect(Collectors.toList());
```